solutions/usaco-941.mdx

```diff
@@ -2,9 +2,73 @@
 id: usaco-941
 source: USACO Bronze 2019 Open
 title: Cow Evolution
-author: Jesse Choe
+author: Jesse Choe, Ryan Chou
 ---
 
+## Solution 
 [Official Analysis](http://www.usaco.org/current/data/sol_evolution_bronze_open19.html)
 
 [Video Analysis](https://youtu.be/4YTijr0BbrA)
```

Comment:
```suggestion
## Video Solution
<Youtube id="4YTijr0BbrA" />
```
we have a way to embed videos (idk why this is not mentioned in the MDX module?)